resources/qml/ActionPanel/OutputProcessWidget.qml

```diff
@@ -0,0 +1,121 @@
+// Copyright (c) 2018 Ultimaker B.V.
+// Cura is released under the terms of the LGPLv3 or higher.
+
+import QtQuick 2.7
+import QtQuick.Controls 2.1
+import QtQuick.Layouts 1.3
+
+import UM 1.1 as UM
+import Cura 1.0 as Cura
+
+Column
+{
+    id: widget
+
+    spacing: UM.Theme.getSize("thin_margin").height
+
+    UM.I18nCatalog
+    {
+        id: catalog
+        name: "cura"
+    }
+
+    Item
+    {
+        id: information
+        width: parent.width
+        height: childrenRect.height
+
+        Column
+        {
+            id: timeAndCostsInformation
+            spacing: UM.Theme.getSize("thin_margin").height
+
+            anchors
+            {
+                left: parent.left
+                right: printInformationPanel.left
+                rightMargin: UM.Theme.getSize("thin_margin").height
+            }
+
+            Cura.IconLabel
+            {
+                id: estimatedTime
+                width: parent.width
+
+                property var printDuration: PrintInformation.currentPrintTime
+
+                text: printDuration.getDisplayString(UM.DurationFormat.Long)
+                source: UM.Theme.getIcon("clock")
+                font: UM.Theme.getFont("small")
+            }
+
+            Cura.IconLabel
+            {
+                id: estimatedCosts
+                width: parent.width
+
+                property var printMaterialLengths: PrintInformation.materialLengths
+                property var printMaterialWeights: PrintInformation.materialWeights
+
+                function getText()
```

Comment:
This needs a better name.

Reply:
Also; since you're only using it once, you could just put the content in of the function in the text property;
text: { var totalLengths = 0 , etc }

Reply:
I wrote it in a different function for readability. I prefer to have a separate function that does something more than a line in a function, but if you don't like it, I don't have a problem in changing it 😃 

Done in 2dfd354b25e3187f403cc1cffc028d61eb7bae2e